src/apps/appstore/app.js

```diff
@@ -13,18 +13,74 @@ define(function(require) {
 			'fr-FR': { customCss: false },
 			'ru-RU': { customCss: false }
 		},
+		requests: {
+			'marketplace.get': {
+				url: 'apps_store/marketplace',
+				verb: 'GET'
+			},
+			'marketplace.action': {
+				apiRoot: monster.config.api.default,
+				url: 'apps_store/marketplace',
+				verb: 'PATCH'
+			}
+		},
+
+		/**
+		 * Store getter
+		 * @param  {Array|String} [path]
+		 * @param  {*} [defaultValue]
+		 * @return {*}
+		 */
+		getStore: function(path, defaultValue) {
+			var self = this,
+				store = ['_store'];
+			return _.get(
+				self,
+				_.isUndefined(path)
+					? store
+					: _.flatten([store, _.isString(path) ? path.split('.') : path]),
+				defaultValue
+			);
+		},
+
+		/**
+		 * Store setter
+		 * @param  {Array|String|*} path|value
+		 * @param  {*} [value]
+		 */
+		setStore: function(path, value) {
+			var self = this,
+				hasValue = _.toArray(arguments).length === 2,
+				store = ['_store'];
+			_.set(
+				self,
+				hasValue
+					? _.flatten([store, _.isString(path) ? path.split('.') : path])
+					: store,
+				hasValue ? value : path
+			);
+		},
```

Comment:
Just making it clear that those methods are [exposed by default](https://github.com/2600hz/monster-ui/blob/5.1/src/js/lib/monster.apps.js#L436) on both `master` and `5.1`.

Copy/pasting them outright should only be done if there is plan to cherry-pick this PR's changes into 5.0.

Even then, there is point to be made so that changes targeting 5.1 use 5.1+ features and if backporting is needed, that port gets adapted to whatever version it targets, but that's just my opinion.